src/markdown-pages/docs/cli-command-reference/runs-get.md

```diff
@@ -3,30 +3,146 @@ path: "/docs/cli-command-reference/cli-runs-get"
 - `--bootstrap` is the URL of the ecosystem's bootstrap properties
+- `--format` is the format in which you want the results to display
 
 
 Results are returned on the terminal in the following example format:
 
 ```
-galasactl runs get --runname U456 
-RunName Status   Result ShortTestName
-U456    Finished Passed MyTestName
+galasactl runs get --name U456 --format details
+name            :  U456
+status          :  Finished
+result          :  Passed
+submitted-time  :  2023-05-04 10:55:29
+start-time      :  2023-05-05 06:00:14
+end-time        :  2023-05-05 06:00:15
+duration(ms)    :  1000
+test-name       :  dev.galasa.Zos3270LocalJava11Ubuntu
+requestor       :  galasa
+bundle          :  dev.galasa
+run-log         :  https://127.0.0.1/ras/run/cbd-123/runlog
+
+method          type status   result start-time          end-time            duration(ms)
+testCoreIvtTest test finished passed 2023-05-05 06:03:38 2023-05-05 06:03:39 1000
 ```
 
+## View tests results in raw format
+
+The following example command returns test status in a raw format:
+
+On Mac or Unix:
+
+```
+galasactl runs get --name U456 --bootstrap http://example.com:30960/boostrap \
+--format raw
+``` 
+
+On Windows (Powershell):
+```
+galasactl runs get --name U456 --bootstrap http://example.com:30960/boostrap `
+--format raw
+``` 
 
+where:
+- `--name` is the name of the test run 
+- `--bootstrap` is the URL of the ecosystem's bootstrap properties
+- `--format` is the format in which you want the results to display
 
+Results are returned on the terminal in the following example format:
 
+```
+$galasactl runs get --name U456 --format raw
+U456|Finished|Passed|2023-05-04T10:55:29.545323Z|2023-05-05T06:00:14.496953Z|2023-05-05T06:00:15.654565Z|1157|dev.galasa.Zos3270LocalJava11Ubuntu|galasa|dev.galasa|https://127.0.0.1/ras/run/cbd-123/runlog
+```
 
+## Query test results by a specific requestor over a given time period
 
+The following examples retrieve a list of test results between two times for tests that were submitted to the ecosystem by a specified requestor.
 
+The following command queries runs that were submitted by requester `johnsmith` from two weeks ago and returns the results in summary format.
+
+```
+galasactl runs get --requestor johnsmith --age 2w
+```
+
+where:
+
+- `--requestor` specifies the requestor name that is associated with the test runs that were submitted to the ecosystem
+- `--age` specifies the time period in which the tests ran
+
+
+The following command queries runs that were submitted by requester `janebrown` from four days ago to two days ago and returns the results in details format.
+
+```
+galasactl runs get --requestor janebrown --age 4d:2d --format details
+```
+
+where:
 
+- `--requestor` specifies the requestor name that is associated with the test runs that were submitted to the ecosystem
+- `--age` specifies the time period in which the tests ran
+- `--format` is the format in which you want the results to display
```

Comment:
we should probably have an example output here as well? maybe

Reply:
have updated an example in summary format